starcheck/src/starcheck

```diff
@@ -3,12 +3,12 @@
 unset LD_LIBRARY_PATH
 unset SYBASE
 unset PYTHONPATH
-SKA_bin=`dirname $0`
-eval `${SKA_bin}/flt_envs -shell sh`
+eval `${SKA}/bin/flt_envs -shell sh`
 export PYTHONHOME=$SKA_ARCH_OS
+export SCDIR=${SKA_ARCH_OS}/lib/python2.7/site-packages/starcheck
```

Comment:
Good idea.  I was hung up on the fact that $STARCHECK is the name of where starcheck.pl puts everything, but that isn't a real collision for this separate shell script.